### ISSUE TITLE
Auto-follow ideas by category.

### ISSUE THREAD

**Issue by mezis** [mezis/appfab#6]
Let users "subscribe" to categories, and have them auto-follow all new ideas in said categorie(s).

Particularly useful for PMs.

**Comment by maxhousetrip** [comment]
User would subscribe to categories in their user profile as part of their joining email that we send manually to a team such as the HAM team.

**Comment by mezis** [comment]
@maxhousetrip agreed, which makes this feature dependent on "invite users" (#34). That feature is however in the next milestone (_the wanderer_), shall I move this one there or the other way around?
